include/crm/stonith-ng.h

```diff
@@ -482,11 +483,11 @@ time_t stonith_api_time(uint32_t nodeid, const char *uname, bool in_progress);
 
 #  define STONITH_LIBRARY "libstonithd.so.26"
 
-typedef int (*st_api_kick_fn) (int nodeid, const char *uname, int timeout, bool off);
+typedef int (*st_api_kick_fn) (int nodeid, const char *uname, int timeout, bool off, int delay);
 typedef time_t (*st_api_time_fn) (int nodeid, const char *uname, bool in_progress);
 
 static inline int
-stonith_api_kick_helper(uint32_t nodeid, int timeout, bool off)
+stonith_api_kick_helper(uint32_t nodeid, int timeout, bool off, int delay)
```

Comment:
Hmm, I'm reluctant to do this since DLM uses this. We'd have to get them to update (while remaining compatible with older pacemakers), and they would gain a version dependency to manage. Is there a good use case for an external app to add a delay anyway?

Reply:
Actually not that I am aware of yet. I was also hesitant about this.  Since we are seeking to avoid bump of the library version, it makes to keep it as it is.